draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2498,17 +2544,30 @@ http://example.com/hls/35cdc008/audio/index.m3u8\n”,
            held by the dCDN based on the objects' URLs.
         </t>
 
-         <t>Spec object specification<list style="empty">
+	<t>URL spec is encoded as a JSON object containing the following name/value pairs:</t>
```

Comment:
Hmm, the title is "URLs Spec", the description calls it "urls spec", and this calls it "URL spec".  Maybe we should make this consistent?

Reply:
Note: I might have missed a newer commit for "urls spec", but "URL spec" still needs to be fixed.